layouts/services/single.html

```diff
@@ -72,13 +72,35 @@ <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl">{{ .services.title }}</h3>
         </div>
         {{ end }}
 
-        {{ if .action.enable }}
-        <div class="mt-4 text-base sm:mt-5 lg:mt-14 lg:text-lg">
-          {{ .action.description | markdownify }}
-        </div>
+        <!-- Case Studies -->
+        {{ if $data.casestudies.casestudies.enable }}
+        {{ with $data.casestudies.casestudies }}
+        <section class="my-8 ">
+            <h2 class="mb-4 font-bold text-xl sm:text-2xl lg:text-3xl">{{ .title }}</h2>
+            <p class="md:text-xl">{{ .subtitle }}</p>
+            <div class="grid md:grid-cols-2 lg:grid-cols-3 gap-8 mt-8 text-center">
+                {{ range .studies }}
+                <div class="bg-[#1D65A6] pb-4 rounded-md text-[#FCFCFC]">
+                    <div>
+                        <h4 class="text-lg md:text-xl font-bold my-6">{{ .client_name}}</h4>
+                        <p class="md:text-lg mx-4 pb-4 md:pb-0 md:h-64">{{ .summary }}</p>
+                    </div>
+                    <a href="{{ .link }}" class="flex justify-center items-center md:text-lg font-bold">
```

Comment:
I think that I may make the entire box clickable instead of the `Learn More` text.